t/unit/contrib/proj/foo.py

```diff
@@ -1,7 +1,7 @@
 from __future__ import absolute_import, unicode_literals
 
 from celery import Celery, shared_task
-from xyzzy import plugh  # noqa
+# from xyzzy import plugh  # noqa
```

Comment:
Why is this commented out?

Reply:
When running `pytest t/unit/contrib/test_pytest.py`, I see this error:

```python
    from __future__ import absolute_import, unicode_literals

    from celery import Celery, shared_task
>   from xyzzy import plugh  # noqa
E   ModuleNotFoundError: No module named 'xyzzy'
```

I understand it might be working in the CI jobs, so I will comment it in, again.